Tasks/KubernetesV1/npm-shrinkwrap.json

```diff
@@ -529,46 +489,37 @@
       }
     },
     "vsts-task-lib": {
```

Comment:
The task shouldn't have reference to `vsts-task-lib` right?

Reply:
vsts-task-tool-lib has an internal reference to vsts-task-lib